rails_docker/Gemfile.txt

```diff
@@ -45,8 +47,9 @@ group :development, :test do
   gem 'rspec-rails' # Rails testing engine
   gem 'guard-rspec' # Auto-run specs
   gem 'shoulda-matchers' # Tests common Rails functionalities
-  gem 'capybara' # Integration testing
+  gem 'capybara', '>= 2.15' # Integration testing
   gem 'selenium-webdriver', '3.7.0' # Ruby bindings for Selenium/WebDriver
+  gem 'chromedriver-helper' # Easy installation and use of chromedriver to run system tests with Chrome
```

Comment:
We don't need this gem @rosle, we will use the Chrome driver that was installed in the Dockerfile

Reply:
@anduonghien @rosle I will create another issue to clean up the default gems in the Gemfile as there are lots we don't need anymore. SO this can be done here in another PR.

Reply:
Yeah, but we can remove this gem from here too because it's a new gem, and to clean up the Gemfile, we will do it in another PR

Reply:
Agreed. Let's do it here. New issue for the other gems: https://github.com/nimblehq/rails-templates/issues/89

Reply:
@anduonghien @olivierobert But on the development now, we're not running with docker, correct?. But I think I can remove it for now because currently we install it locally, right?. 

I will remove it in this pr. We can revisit the setup later in another pr 👍

Reply:
On development, I think we don't need the Google chrome driver 🤔 we need it only for test, I think